### PR TITLE
Snap fix

### LINKED ISSUE #435: Remote snapshots are not deleted
**Describe the bug**
purefa_snap unable to destroy/eradicate remoe snapshots

**To Reproduce**
1. Create a remote snapshot
2. Destroy and eradicate remote snapshot from target array
3. Verify the snapshot still exists on target array

**Expected behavior**

Remote snapshot gets deleted

** Additional notes **
The problem is due to a possible issue with API 1.x, in which the volume search method returns an error if a remote snapshot is specified.

### PR DESCRIPTION
**Pull request by genegr**
##### SUMMARY

Fixed #435 

##### ISSUE TYPE
- Bugfix Pull Request

##### COMPONENT NAME
purefa_snap

### CODE REVIEW

**Review by sdodsley — DISMISSED** [review]
No position to test but I assume you have tested multiple scenarios...

**Comment by sdodsley** [comment]
@genegr please add a release note

**Comment by genegr** [comment]
Yep, tested all the combinations of automatically and user assigned postfix
and it worked. It is mainly a workaround to the response error returned by
the volumes API 1.x. A normal query for a remote snapshot returns an error
with the specific reason that is a remote volume snapshot, while with the
wildcard '*' it matches all the snapshots for the volumes, without taking
into consideration they are remote or local.

**Comment by genegr** [comment]
Done. Added the proper changelog fragment file

**Comment by sdodsley** [comment]
Changelog fragments needs to be a .yml fe

File: changelogs/fragments/436_snap_fix.yaml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+bugfixes:
+ - purefa_snap - Fixed issue with remote snapshot retrieve. Mainly a workaround to an issue with Purity REST 1.x when remote snapshots are serached
```
Comment (sdodsley) [inline changelogs/fragments/436_snap_fix.yaml:2]: Spelling

Comment (genegr) [reply changelogs/fragments/436_snap_fix.yaml]: Amended

**Review by sdodsley — APPROVED** [review]

**Pull request merged**